app/src/pages/inside/projectSettingsPageContainer/content/notifications/modals/addEditNotificationModal/addEditNotificationModal.jsx

```diff
@@ -0,0 +1,374 @@
+    id: 'AddEditNotificationCaseModal.dropdownValueAlways',
+    defaultMessage: 'Always',
+  },
+  [LAUNCH_CASES.MORE_10]: {
+    id: 'AddEditNotificationCaseModal.dropdownValueMore10',
+    defaultMessage: '> 10% of items have issues',
+  },
+  [LAUNCH_CASES.MORE_20]: {
+    id: 'AddEditNotificationCaseModal.dropdownValueMore20',
+    defaultMessage: '> 20% of items have issues',
+  },
+  [LAUNCH_CASES.MORE_50]: {
+    id: 'AddEditNotificationCaseModal.dropdownValueMore50',
+    defaultMessage: '> 50% of items have issues',
+  },
+  [LAUNCH_CASES.FAILED]: {
+    id: 'AddEditNotificationCaseModal.dropdownValueFailed',
+    defaultMessage: 'Launch has issues',
+  },
+  [LAUNCH_CASES.TO_INVESTIGATE]: {
+    id: 'AddEditNotificationCaseModal.dropdownValueToInvestigate',
+    defaultMessage: 'Launch has "To Investigate" items',
+  },
+  controlPanelName: {
+    id: 'AddEditNotificationCaseModal.controlPanelName',
+    defaultMessage: 'Rule',
+  },
+  noItemsMessage: {
+    id: 'NoCasesBlock.noItemsMessage',
+    defaultMessage: 'No Email Notification Rules',
+  },
+  notificationsInfo: {
+    id: 'NoCasesBlock.notificationsInfo',
+    defaultMessage: 'Once a launch is finished system will notify selected people by email',
+  },
+  create: {
+    id: 'AddNewCaseButton.addNewRuleButton',
+    defaultMessage: 'Create Rule',
+  },
+  toggleLabel: {
+    id: 'NotificationsEnableForm.toggleNotificationsLabel',
+    defaultMessage: 'E-mail notification',
+  },
+  toggleNote: {
+    id: 'NotificationsEnableForm.toggleNotificationsNote',
+    defaultMessage: 'Send e-mail notifications about launches finished',
+  },
+});
+
+const AddEditNotificationModal = ({
+  data,
+  data: { isNewCase, onSave },
+  handleSubmit,
+  initialize,
+}) => {
+  const { formatMessage } = useIntl();
+  const { trackEvent } = useTracking();
+  const dispatch = useDispatch();
+
+  const activeProject = useSelector(projectIdSelector);
+  const [isEditorShown, setShowEditor] = React.useState(false);
+
+  useEffect(() => {
+    initialize(data.notification);
+  }, []);
+
+  const getDropdownInputConfig = () => {
+    return [
+      {
+        value: LAUNCH_CASES.ALWAYS,
+        label: formatMessage(messages[LAUNCH_CASES.ALWAYS]),
+      },
+      {
+        value: LAUNCH_CASES.MORE_10,
+        label: formatMessage(messages[LAUNCH_CASES.MORE_10]),
+      },
+      {
+        value: LAUNCH_CASES.MORE_20,
+        label: formatMessage(messages[LAUNCH_CASES.MORE_20]),
+      },
+      {
+        value: LAUNCH_CASES.MORE_50,
+        label: formatMessage(messages[LAUNCH_CASES.MORE_50]),
+      },
+      {
+        value: LAUNCH_CASES.FAILED,
+        label: formatMessage(messages[LAUNCH_CASES.FAILED]),
+      },
+      {
+        value: LAUNCH_CASES.TO_INVESTIGATE,
+        label: formatMessage(messages[LAUNCH_CASES.TO_INVESTIGATE]),
+      },
+    ];
+  };
+
+  const okButton = {
+    text: formatMessage(COMMON_LOCALE_KEYS.SAVE),
+    onClick: () => handleSubmit(onSave)(),
```

Comment:
it's not working. Click on save button do nothing right now

Reply:
Fixed